core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -120,6 +120,11 @@ class CudaInternal {
   bool was_initialized = false;
   bool was_finalized   = false;
 
+  // these want to be per-device, not per-stream...  use of 'static' here will
+  //  break once there are multiple devices though
```

Comment:
Please add a FIXME or FIXME_CUDA so that we can easily grep for this comment.

Reply:
ok - update coming momentarily